tests/unit/utils/files.test.ts

```diff
@@ -0,0 +1,20 @@
+import { describe, expect, it } from "vitest";
+
+import { DateLevel, serializeDateForFileName } from "../../../src/utils/files.js";
+
+describe("serializeDateForFileName", () => {
+  it.each([
+    ["2025", DateLevel.Year],
+    ["2025-02", DateLevel.Month],
+    ["2025-02-07", DateLevel.Day],
+    ["2025-02-07-15", DateLevel.Hour],
+    ["2025-02-07-15-08", DateLevel.Minute],
+    ["2025-02-07-15-08-40", DateLevel.Second],
+  ])(
+    "correctly serializes datetime",
```

Comment:
Include the case data in the test-case name so the cases are distinguishable. Now it is not possible to know which case failed. You can use there printf formatting. https://vitest.dev/api/#test-each